dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   string_scanner: 1.1.0
   url_launcher: 6.0.17
   cupertino_icons: 1.0.4
-  video_player: 2.1.1
+  video_player: 2.2.11
```

Comment:
Weird, this totally exists, and it works for me locally.
```
ERR : Because flutter_gallery depends on video_player 2.2.11 which doesn't match any versions, version solving failed.
```

Also, run `flutter update-packages --force-upgrade`, you'll see the autogenerated `video_player_web` line get bumped too.

Reply:
I don't see the reasoning in #83367 or #82275.  @zanderso do you remember?

Reply:
@blasten does 2.1.2 work?

Reply:
2.2.11 worked after you ran `update-packages`.

Reply:
I don't remember why it was pinned.